src/vs/workbench/services/configurationResolver/common/configurationResolver.ts

```diff
@@ -57,7 +57,7 @@ export interface CommandInputInfo {
 	id: string;
 	type: 'command';
 	command: string;
-	args?: any;
+	args?: any[];
```

Comment:
Changing args to be an array breaks all old launch configs not using arrays. So we will have to support both `any | any[]`.